README.md

```diff
@@ -1 +1,39 @@
-# Portfolio
\ No newline at end of file
+# My-Portfolio
+This is my second microverse project. In this project I built my portfolio(Mobile version).Thismarks the second step for the portfolio project.The main objective for this project was to build template from figma using CSS and Flexbox. I created my-portfolio reposotory and cloned into my local machine. I have implemented linter,Github Flow and professional documentation which is one of the project building requirements at microverse.
+
```

Comment:
- [ ] As presented in the template please make the font-weight of the different section titles - 'My Recent Works', 'Multi-Post Stories' and 'Profesional Art Printing Data' - should be **bolder** than they are.

![header-styles](https://user-images.githubusercontent.com/36542986/134500356-612497a0-0c0f-461d-93e5-e97ff1bcf64b.jpg)